# HG changeset patch
# User pcxfirefox@gmail.com
# Date 1396626649 -28800
#      Fri Apr 04 23:50:49 2014 +0800
# Node ID 2a231aef2418a25cd43bc0d2faac964aa166be5a
# Parent  4ef6cc9f583f6331309765ce5ae6be3bfe258be2
Remove archivereader support (  Blob support for Zip file contents )

diff -r 4ef6cc9f583f -r 2a231aef2418 dom/bindings/Bindings.conf
--- a/dom/bindings/Bindings.conf	Fri Apr 04 23:28:57 2014 +0800
+++ b/dom/bindings/Bindings.conf	Fri Apr 04 23:50:49 2014 +0800
@@ -101,14 +101,6 @@
     'nativeType': 'nsDOMAnimationEvent',
 },
 
-'ArchiveReader': {
-    'nativeType': 'mozilla::dom::file::ArchiveReader',
-},
-
-'ArchiveRequest': {
-    'nativeType': 'mozilla::dom::file::ArchiveRequest',
-},
-
 'AudioChannelManager': {
     'nativeType': 'mozilla::dom::system::AudioChannelManager',
     'headerFile': 'AudioChannelManager.h'
diff -r 4ef6cc9f583f -r 2a231aef2418 dom/file/moz.build
--- a/dom/file/moz.build	Fri Apr 04 23:28:57 2014 +0800
+++ b/dom/file/moz.build	Fri Apr 04 23:50:49 2014 +0800
@@ -18,11 +18,6 @@
 ]
 
 EXPORTS.mozilla.dom.file += [
-    'ArchiveEvent.h',
-    'ArchiveReader.h',
-    'ArchiveRequest.h',
-    'ArchiveZipEvent.h',
-    'ArchiveZipFile.h',
     'DOMFileHandle.h',
     'DOMFileRequest.h',
     'File.h',
@@ -34,11 +29,6 @@
 ]
 
 UNIFIED_SOURCES += [
-    'ArchiveEvent.cpp',
-    'ArchiveReader.cpp',
-    'ArchiveRequest.cpp',
-    'ArchiveZipEvent.cpp',
-    'ArchiveZipFile.cpp',
     'AsyncHelper.cpp',
     'DOMFileHandle.cpp',
     'DOMFileRequest.cpp',
diff -r 4ef6cc9f583f -r 2a231aef2418 dom/webidl/moz.build
--- a/dom/webidl/moz.build	Fri Apr 04 23:28:57 2014 +0800
+++ b/dom/webidl/moz.build	Fri Apr 04 23:50:49 2014 +0800
@@ -20,8 +20,6 @@
     'AnimationEvent.webidl',
     'AppInfo.webidl',
     'AppNotificationServiceOptions.webidl',
-    'ArchiveReader.webidl',
-    'ArchiveRequest.webidl',
     'Attr.webidl',
     'AudioBuffer.webidl',
     'AudioBufferSourceNode.webidl',
